libraries/AP_Scripting/examples/LED_poslight.lua

```diff
@@ -0,0 +1,83 @@
+--[[
+ Script to use LED strips as position lights.
+--]]
+
+--[[
+ For this script we will use two strips with up to 8 LEDs each.
+--]]
+local num_leds = 8
+
+local timer = 0
+
+--[[
+ Brightness for green or red light.
+--]]
+local br_color = 255
+
+--[[
+ Brightness for flash light when armed.
+--]]
+local br_flash = 255
+
+--[[
+ Use SERVOn_FUNCTION 94 for left LED strip 
+ Use SERVOn_FUNCTION 95 for right LED strip
+--]]
+local chan_left = SRV_Channels:find_channel(94)
+local chan_right = SRV_Channels:find_channel(95)
+
+if not chan_left then
+    gcs:send_text(6, "LEDs left: channel not set")
+    return
+end
+
+if not chan_right then
+    gcs:send_text(6, "LEDs right: channel not set")
+    return
+end
+
+-- find_channel returns 0 to 15, convert to 1 to 16
+chan_left = chan_left + 1
+chan_right = chan_right + 1
+
+gcs:send_text(6, "LEDs strip left: chan=" .. tostring(chan_left))
+gcs:send_text(6, "LEDs strip right: chan=" .. tostring(chan_right))
+
+-- initialisation code
+serialLED:set_num_neopixel(chan_left, num_leds)
```

Comment:
Probably worth also asserting these, that can return false.
```suggestion
assert(serialLED:set_num_neopixel(chan_left, num_leds),"Failed left LED setup")
```

Reply:
Thanks, done.